### PR TITLE
Include Waveshare RP2040 One (3 of 3)

### PR DESCRIPTION
**Pull request by johnnohj**
Part of a trio of PRs to see support for the RP2040 One extended; I think it could be useful as an inexpensive entry to testing out sensors or debugging in light of articles such as:

https://learn.adafruit.com/circuitpython-libraries-on-any-computer-with-raspberry-pi-pico

Companion pull requests:
https://github.com/adafruit/u2if/pull/11

### CODE REVIEW

File: adafruit_platformdetect/board.py [hunk 1]
```diff
@@ -30,7 +30,7 @@
 
 from adafruit_platformdetect.constants import boards, chips
 
-__version__ = "0.0.0+auto.0"
+__version__ = "3.72.1"
```
Comment (makermelissa) [inline adafruit_platformdetect/board.py:33]: Please change this back to 0.0.0+auto.0. The versioning is automatically changed this way.

File: adafruit_platformdetect/chip.py [hunk 2]
```diff
@@ -29,7 +29,7 @@
 
 from adafruit_platformdetect.constants import chips
 
-__version__ = "0.0.0+auto.0"
+__version__ = "3.72.1"
```
Comment (makermelissa) [inline adafruit_platformdetect/chip.py:32]: Please change this back to 0.0.0+auto.0 as well.

File: adafruit_platformdetect/chip.py [hunk 3]
```diff
@@ -109,31 +109,40 @@ def id(
...
-                            0x00F1,
-                            0x00FD,
-                            0x00F7,
-                            0x0109,
-                            0x0107,
-                            0x812C,
-                            0x812E,
-                            0x8130,
-                            0x0105,
+                        (
```
Comment (makermelissa) [inline adafruit_platformdetect/chip.py:114]: Can we remove the extra set of parenthesis?

**Review by makermelissa — CHANGES_REQUESTED** [review]
Thanks for adding. Just a few minor changes requested.

**Comment by johnnohj** [comment]
Checks are stumbling over Black formatting, which insists on the extra level of parentheses wrapping the (now) three groups of vendor/product ids

**Comment by makermelissa** [comment]
> Checks are stumbling over Black formatting, which insists on the extra level of parentheses wrapping the (now) three groups of vendor/product ids

Fair enough. We can leave it then.

**Review by makermelissa — APPROVED** [review]
Looks good. thank you!

**Pull request merged**